### PR TITLE
Added Marketing doc and Product Marketing Manager role

### PR DESCRIPTION
**Pull request by robpearson**
Added Product Marketing Manager (Octopus Release Champion) role.

### CODE REVIEW

File: Marketing.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Marketing
+
+## Product Marketer Manager (Developer doing Marketing)
+
+> I am the voice of the customer, and I am the glue between engineering and marketing to drive interest and people trying our product
+
+**I have a deep knowledge of the ethos, patterns and principles of Octopus Deploy and I understand its benefits for teams and the problems it solves**
+
+* I am the authority on our customers, who are they, how they buy and why they pick Octopus (buying criteria).
```
Comment (michaelnoonan) [inline Marketing.md]: Consider: I am an authority on our customers being a developer myself, and understand what motivates them to choose Octopus

File: Marketing.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* I understand the competitive landscape, and I work to be an expert on our competition and how they are positioned.
+* I collaborate with engineering teams and marketing to develop product messaging and positioning that resonates with our customers (target personas).
+* I understand and document our customer’s buying process, including where they get information, and the who, what, when and why behind the decisions they make. Then drive changes to our sales and marketing processes based on what you learn.
```
Comment (michaelnoonan) [inline Marketing.md]: I help dig beneath the surface of the evidence in our marketing tools to help our marketing team understand the real buying process?

File: Marketing.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Marketing
+
+## Developer doing Marketing (Product Marketer Manager)
+
+> I am the voice of the customer, and I am the glue between engineering and marketing to drive interest and people trying our product
+
+**I have a deep knowledge of the ethos, patterns and principles of Octopus Deploy and I understand its benefits for teams and the problems it solves**
+
+* I am the authority on our customers, who are they, how they buy and why they pick Octopus (buying criteria).
```
Comment (tothegills) [inline Marketing.md]:
```suggestion
* I am the authority on our customers: who they are, how they buy and why they pick Octopus (buying criteria).
```

Comment (robpearson) [reply Marketing.md]: This is great. Updating!

File: Marketing.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+**I am the glue between engineering and marketing** 
+
+* I help the engineering team understand the value of marketing activities and work with them to plan feature launches, messaging and content.
+* I help the marketing team understand software development, DevOps and deployment automation and their value
+* I help the marketing team confidently update Octopus.com via code updates, pull requests, building and deploying.
+
+**I oversee the content we produce, and I ensure it's right for our audience** 
+
+* I am the gatekeeper for content (written, video, speaking etc.) and I ensure it's appropriate for our audience
+* I ensure we communicate inline with the Octopus Personality, i.e. We're friendly, human, genuine etc.
```
Comment (michaelnoonan) [inline Marketing.md]: and you can't "smell the marketing"

Comment (robpearson) [reply Marketing.md]: That fits perfectly!

File: Marketing.md (hunk 2 above)
Comment (robpearson) [reply Marketing.md]: That's a great update to the last statement which I got off a sample job description. It's a lot more readable/human.

**Pull request merged**